### DISCUSSION TITLE
How to rollback to 16.1.0 versionfrom 18.0.0

### DISCUSSION THREAD

**Discussion by rsreddy1992** [keycloak/keycloak#11970] [Archive]
Hi, we are using keyclaok, when pods got restarted version automatically updated from 16.1.0 to 18.0.0  how do we rollback to 16.1.0 version?

**Comment by andreaTP** [comment]
Hi @rsreddy1992 , would you mind elaborating on your setup? (e.g. are you using the operator? the new preview one? etc. etc.)

Both the old and new operators are supporting 1-1 version compatibility, so, if you install the operator shipped at version `18.0.0` you will be able to install only Keycloak `18.0.0` by default.

**Comment by rsreddy1992** [comment]
Hi, yes, we are using the olm operator.i am trying to rollback but it is not happening

**Reply by andreaTP** [reply]
Hi @rsreddy1992 , I'm sorry to say that this is, by far, not enough information to help you out.

Please provide the following:
 - details on your target infrastructure (Kubernetes version, OLM version, etc. etc.)
 - a dump of the `Subscription` CR for Keycloak in the cluster (and check the `.status` field)
 - possibly reproduction steps (e.g. using `minikube`)
